packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -181,8 +199,18 @@ class CupertinoButton extends StatefulWidget {
   ///
   /// Defaults to kMinInteractiveDimensionCupertino which the iOS Human
   /// Interface Guidelines recommends as the minimum tappable area.
+  @Deprecated(
+    'Use minimumSize instead. '
+    'This feature was deprecated after v3.28.0-3.0.pre.',
+  )
   final double? minSize;
 
+  /// The minimumSize of the button.
+  ///
+  /// Defaults to [kMinInteractiveDimensionCupertino], which the iOS Human
```

Comment:
Nit: a `Size` can't default to a `double`. Maybe something like:

```suggestion
  /// Defaults to a button with a height and a width of [kMinInteractiveDimensionCupertino], which the iOS Human
```